### PR TITLE
Add create fulfillment webhook

### PR DESCRIPTION
**Pull request by szewczykmira**
This PR introduces webhook that is used for integration with external shipping.

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] Database migration files are up to date.
1. [ ] The changes are tested.
1. [ ] GraphQL schema and type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### CODE REVIEW

**Comment by django-queries** [comment]
Here is the report for 1f5587cbd1b77cb34fe95602ec352e884b3d1318 (mirumee/saleor @ feature/fulfillment-webhook)
Base comparison is 00284d0c65eb063c339fc69366bbbc21ac3955d3.



<details><summary>No differences were found. (click me)</summary>
<p>

```diff
# api.benchmark checkout
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  add billing address to checkout            	         34	         34	             20
  add shipping to checkout                   	          7	          7	              0
  checkout payment charge                    	         10	         10	              0
  complete checkout                          	          8	          8	              0
  create checkout                            	          5	          5	              1

# api.benchmark homepage
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  retrieve main menu                         	          5	          5	              0
  retrieve product list                      	          4	          4	              0
  retrieve secondary menu                    	          5	          5	              0
  retrieve shop                              	          2	          2	              0

# api.benchmark product
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product details                            	         18	         18	              4
  retrieve product attributes                	          9	          9	              0

# api.benchmark variant
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product variant bulk create                	         51	         51	              3
  retrieve variant list                      	         23	         23	              9

# api product sorting attributes
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  sort product not having attribute data     	         21	         21	              0
```

</p>
</details>

File: saleor/order/actions.py [hunk 1]
```diff
@@ -111,11 +111,13 @@ def order_fulfilled(
 ):
     order = fulfillment.order
     update_order_status(order)
+    print(user)
```
Comment (maarcingebala) [inline saleor/order/actions.py]: Sir, please remove this debug code. Thank you, Sir.

File: saleor/webhook/payloads.py [hunk 2]
```diff
@@ -208,6 +210,45 @@ def generate_product_payload(product: "Product"):
...
+    serializer = PayloadSerializer()
+    line_fields = ("quantity",)
+
+    return serializer.serialize(
+        fulfillment,
+        fields=line_fields,
+        extra_dict_data={
+            "weight": (lambda fl: fl.order_line.variant.get_weight().oz),
+            "product_type": (
+                lambda fl: fl.order_line.variant.product.product_type.name
```
Comment (maarcingebala) [inline saleor/webhook/payloads.py]: Did you test this code for database performance? It requires a few joins so I would just make sure that we have proper prefetching where it's necessary.

Comment (korycins) [reply saleor/webhook/payloads.py]: I've already added prefetch here.  5 queries vs 89

**Pull request merged**